package/yast2-schema-default.spec

```diff
@@ -31,7 +31,7 @@ Url:            https://github.com/yast/yast-schema
 
 Provides:       yast2-schema
 # others cannot be used as it contains same files
-Conflicts:      otherproviders(yast2-schema)
+Conflicts:      yast2-schema
```

Comment:
well, it was requested by @jsrain https://github.com/yast/yast-schema/pull/126/files#r791805872